components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java

```diff
@@ -1012,6 +1018,24 @@ private JSExecutionMonitorData endScriptExecutionMonitor(String identifier) {
         return getJSExecutionSupervisor().completed(identifier);
     }
 
+    private void setCurrentStepAsSubjectIdentifier(StepConfig stepConfig) {
+        stepNamedMap.forEach((integer, config) -> { // remove existing subject identifier step
```

Comment:
A new line before the method body.

Reply:
Comments need to start with capital letter and ends with full stop

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/4068